packages/storybook/stories/va-select-uswds.stories.jsx

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect, useState } from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+
+const selectDocs = getWebComponentDocs('va-select');
+
+export default {
+  title: 'USWDS/Select USWDS',
+  id: 'uswds/va-select',
+  parameters: {
+    componentSubtitle: 'va-select web component',
+    docs: {
+      page: () => <StoryDocs data={selectDocs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  'label': 'Branch of Service',
+  'name': 'branch',
+  'value': 'army',
+  'required': false,
+  'error': undefined,
+  'uswds': true,
+  hint: null,
+  'aria-live-region-text': 'You selected',
+  'options': [
+    <option key="1" value="navy">
+      Navy
+    </option>,
+    <option key="2" value="army">
+      Army
+    </option>,
+    <option key="3" value="marines">
+      Marines
+    </option>,
+    <option key="4" value="air-force">
+      Air Force
+    </option>,
+    <option key="5" value="coastguard">
+      Coastguard
+    </option>,
+  ],
+  'use-add-button': false,
+};
+
+const Template = ({
+  label,
+  name,
+  value,
+  required,
+  error,
+  uswds,
+  hint,
+  'aria-live-region-text': ariaLiveRegionText,
```

Comment:
I'm guessing that this `aria-live-region-text` was added to the story because the original `va-select.stories.jsx` file has it? 

But I'm a little confused about it. It's not a prop in the component itself. It just seems to be added as some kind of guidance in Storybook. And when I research the purpose of it, I can't seem to find any documentation about having this type of aria attribute. Also when I test it with VoiceOver, it does not seem to announce anything when the select changes. 

I'm going to reach out to the #accessibility-help channel to see if we can determine if this is necessary or not.

Reply:
@jhonnyoddball Based on this [feedback from Josh](https://dsva.slack.com/archives/C8E985R32/p1675095738560739), I don't think we should include this `aria-live-region-text` attribute to the v3 Storybook examples. I'm unable to find any documented history about this or able to get it to work in a screenreader and his feedback says that it would be redundent.

I'll write up a separate ticket though to follow-up on this for the `va-select.stories.jsx` Storybook file though once we have an a11y specialist.

Reply:
Follow up ticket: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1483

Reply:
Yeah, I duplicated the file and just added what's needed